ci-kubernetes-e2e-kubeadm-kinder-1-26-on-1-25 failed for nil pointer when removing etcd member

https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/ci-kubernetes-e2e-kubeadm-kinder-1-26-on-1-25/1687241264644231168


```
I0803 23:57:29.524325   11049 etcd.go:350] Member was already removed, because member 40fd14fa28910cab was not found
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x1a1d5fe]
goroutine 1 [running]:
k8s.io/kubernetes/cmd/kubeadm/app/util/etcd.(*Client).RemoveMember(0xc00058e040, 0x40fd14fa28910cab)
	cmd/kubeadm/app/util/etcd/etcd.go:363 +0x11e
k8s.io/kubernetes/cmd/kubeadm/app/phases/etcd.RemoveStackedEtcdMemberFromCluster({0x2230e90, 0xc000602d00}, 0xc00062c000)
	cmd/kubeadm/app/phases/etcd/local.go:129 +0x41e
k8s.io/kubernetes/cmd/kubeadm/app/cmd/phases/reset.runRemoveETCDMemberPhase({0x1ddbbe0?, 0xc0004ece70?})
	cmd/kubeadm/app/cmd/phases/reset/removeetcdmember.go:66 +0x28a
k8s.io/kubernetes/cmd/kubeadm/app/cmd/phases/workflow.(*Runner).Run.func1(0xc0004c78c0)
	cmd/kubeadm/app/cmd/phases/workflow/runner.go:259 +0x134
k8s.io/kubernetes/cmd/kubeadm/app/cmd/phases/workflow.(*Runner).visitAll(...)
	cmd/kubeadm/app/cmd/phases/workflow/runner.go:446
k8s.io/kubernetes/cmd/kubeadm/app/cmd/phases/workflow.(*Runner).Run(0xc0000d53b0, {0xc0005740a0, 0x0, 0x2})
	cmd/kubeadm/app/cmd/phases/workflow/runner.go:232 +0x3e3
k8s.io/kubernetes/cmd/kubeadm/app/cmd.newCmdReset.func1(0xc0006d8000?, {0xc0005740a0?, 0x2?, 0x2?})
	cmd/kubeadm/app/cmd/reset.go:183 +0x26
github.com/spf13/cobra.(*Command).execute(0xc0006d8000, {0xc000574080, 0x2, 0x2})
	vendor/github.com/spf13/cobra/command.go:916 +0x862
github.com/spf13/cobra.(*Command).ExecuteC(0xc000134600)
	vendor/github.com/spf13/cobra/command.go:1040 +0x3bd
github.com/spf13/cobra.(*Command).Execute(0x21fb3c0?)
	vendor/github.com/spf13/cobra/command.go:968 +0x19
k8s.io/kubernetes/cmd/kubeadm/app.Run()
	cmd/kubeadm/app/kubeadm.go:50 +0x15d
main.main()
	cmd/kubeadm/kubeadm.go:25 +0x19
Error: failed to exec action kubeadm-reset: exit status 2
 exit status 1
```

--- a/cmd/kubeadm/app/util/etcd/etcd.go
+++ b/cmd/kubeadm/app/util/etcd/etcd.go
@@ -362,8 +362,11 @@ func (c *Client) RemoveMember(id uint64) ([]Member, error) {
 
 	// Returns the updated list of etcd members
 	ret := []Member{}
-	for _, m := range resp.Members {
-		ret = append(ret, Member{Name: m.Name, PeerURL: m.PeerURLs[0]})
+	if resp != nil {
+		for _, m := range resp.Members {
+			ret = append(ret, Member{Name: m.Name, PeerURL: m.PeerURLs[0]})
+		}
+
 	}
 
 	return ret, nil

